nmigen/compat/genlib/roundrobin.py

```diff
@@ -0,0 +1,50 @@
+from ..._utils import deprecated
+from ..fhdl.module import CompatModule
+
+
+__all__ = ["RoundRobin", "SP_WITHDRAW", "SP_CE"]
+
+(SP_WITHDRAW, SP_CE) = range(2)
+
+@deprecated("instead of `migen.genlib.roundrobin.RoundRobin`, "
+            "use `nmigen.lib.roundrobin.RoundRobin`; note that RoundRobin does not "
+            "require a policy anymore, and that the `ce` attribute has been renamed"
+            "to `en`")
```

Comment:
This comment is outdated. Also, I would explicitly emit two different warning from `__init__` depending on what `switch_policy` is set to.